http-server/http_server_test.go

```diff
@@ -58,50 +74,51 @@ func TestTrustedProxies(t *testing.T) {
 	}
 }
 
-func TestGetMetadataCacher(t *testing.T) {
-	for name, test := range cacherTests {
-		t.Log(name)
-		hegelServer.hardwareClient = hardwareGetterMock{test.json}
-
-		dataModelVersion := os.Getenv("DATA_MODEL_VERSION")
-		defer os.Setenv("DATA_MODEL_VERSION", dataModelVersion)
-		os.Unsetenv("DATA_MODEL_VERSION")
-
-		customEndpoints := os.Getenv("CUSTOM_ENDPOINTS")
-		defer os.Setenv("CUSTOM_ENDPOINTS", customEndpoints)
-		os.Unsetenv("CUSTOM_ENDPOINTS")
-
-		req, err := http.NewRequest("GET", "/metadata", nil)
-		if err != nil {
-			t.Fatal(err)
-		}
-		req.RemoteAddr = mockUserIP
-		resp := httptest.NewRecorder()
-		http.HandleFunc("/metadata", getMetadata("")) // filter not used in cacher mode
-
-		http.DefaultServeMux.ServeHTTP(resp, req)
-
-		if status := resp.Code; status != http.StatusOK {
-			t.Errorf("handler returned wrong status code: got %v want %v",
-				status, http.StatusOK)
-		}
-
-		hw := exportedHardwareCacher{}
-		err = json.Unmarshal(resp.Body.Bytes(), &hw)
-		if err != nil {
-			t.Error("Error in unmarshalling hardware:", err)
-		}
-
-		if hw.ID != test.id {
-			t.Errorf("handler returned unexpected id: got %v want %v",
-				hw.ID, test.id)
-		}
-		if hw.PlanSlug != test.planSlug {
-			t.Errorf("handler returned unexpected plan slug: got %v want %v",
-				hw.PlanSlug, test.planSlug)
-		}
-	}
-}
+// soon to be removed
```

Comment:
why?

Reply:
Aren't we removing the cacher /metadata endpoint?

Reply:
I can put it back for now, I think i just commented it out because i ran into an issue trying to fix that test and I figured since we're removing it soon anyway, I'll not bother

Reply:
oic, no we don't want to remove the /metadata endpoint itself just its current data format should be changed to match Kant's / Hegel's in tink-mode.

Reply:
Not sure if that would help with the tests, if not then yeah we can just delete them and add tests later when /metadata works like legacy with cacher data.

Reply:
Okay, so I'm a little confused. Just to make sure, so if hegel is running in cacher mode, then there won't be a `/metadata` endpoint, is that correct?